geocoder_module/geocoder.py

```diff
@@ -316,6 +316,10 @@ def double_check_countries(
 
         # create a default mapping and extract all the countries
         for location in locations:
+            if not location:
+                continue
+            if not "name" in location:
+                continue
```

Comment:
Feel free to ignore this - but maybe make this one line?
```suggestion
            if not location or not "name" in location:
                continue
```